neo/SmartContract/NeoService.cs

```diff
@@ -165,6 +169,36 @@ public NeoService(TriggerType trigger, Snapshot snapshot)
             #endregion
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="engine">r, s, v, messageHash</param>
+        /// <returns></returns>
+        private bool Ecrecover(ExecutionEngine engine)
+        {
+            var r = new System.Numerics.BigInteger(engine.CurrentContext.EvaluationStack.Pop().GetByteArray().Reverse().Concat(new byte[1]).ToArray());
```

Comment:
I think we should divide `r` and` s` from a `signature` argument, because that is the way we treat the signatures in neo.

Reply:
The problem is that signatures from different chain are different, some will put "v" in front of signature, some will put "v" at the end of it. So will take this design and divide them in contract level

Reply:
You are right, it's good like this